libs/deal-ticket/src/components/market-info/info-market.tsx

```diff
@@ -51,7 +49,10 @@ export const MarketInfoContainer = ({ marketId }: MarketInfoContainerProps) => {
   }, []);
 
   const { data, loading, error } = useQuery(MARKET_INFO_QUERY, {
-    variables: { marketId, interval: Interval.INTERVAL_I1H, since: yTimestamp },
+    variables: useMemo(
```

Comment:
chore (suggestion): can we keep hooks used naturally in the body of the function for readability

Reply:
updated.